routes/spentRoutes.js

```diff
@@ -0,0 +1,38 @@
+var db = require("../models");
```

Comment:
If the two routes will be handled in this one PR I think it will still need some of the suggested changes from the other PR.